app/controllers/navigation/StandardDeclarationJorney.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.navigation
+
+import models.Mode
+import play.api.mvc.Result
+
+import play.api.mvc.Results._
+
+object StandardDeclarationJorney {
```

Comment:
I'm just worried about this object, the size of it when we move here whole journeys, some duplications between Supplementary and Standard

Reply:
This for now is palceholder which hides manual parameter building - I don't know which would i like more - put all logic here and having it spread over views and controllers

Reply:
Same, I don't think we need this object. It is likely to grow large and uses static method references which cant be mocked out in tests